framework/aster-frame/src/trap/handler.rs

```diff
@@ -233,7 +232,12 @@ fn handle_kernel_page_fault(f: &TrapFrame) {
             &(vaddr..vaddr + PAGE_SIZE),
             &(paddr..paddr + PAGE_SIZE),
             MapProperty {
-                perm: VmPerm::RW | VmPerm::G,
+                perm: VmPerm::RW,
+                global: true,
+                #[cfg(feature = "intel_tdx")]
+                extension: PageTableFlags::SHARED.bits() as u64,
+                #[cfg(not(feature = "intel_tdx"))]
+                extension: 0,
```

Comment:
I'm a little curious as to how this change was made. Why change it to share by default?

Pages shouldn't be shared unless explicitly requested, right?

Or just for consistency, the linear mapping built at boot is not shared (at least the last time I checked), right?

Reply:
Oh you have spotted a mistake here. Please open up an issue for it.